crates/test_utils/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "test_utils"
+description = "Postman collection runner and utility"
+version = "0.1.0"
+edition.workspace = true
+rust-version.workspace = true
+readme = "README.md"
+license.workspace = true
+
+[features]
+default = ["dummy_connector"]
+dummy_connector = ["api_models/dummy_connector"]
+external_access_dc = ["dummy_connector"]
```

Comment:
Where is this feature being used?

Reply:
`external_access_dc = ["dummy_connector"]`? No where! Removed.